### DISCUSSION TITLE
v2 Frame View resizing

### DISCUSSION THREAD

**Discussion by tznind** [tui-cs/Terminal.Gui#3524] [Q&A]
@tig regarding the new `FrameView` drag to move.  I am finding that it is currently fighting with the resize in TGD.  Is there a way to disable this feature on the View?

Current issues:

- Resize from bottom right in TGD does not work for FrameView
- In TGD, when dragging a FrameView out of another view (e.g. a TabView) it 'sticks' to the bounds of the TabView.  This behaviour differs from e.g. Button and makes it look like releasing mouse will leave the FrameView where it is.

I also notice that once selected in TGD, Border seems to get stuck as Green.  Although this is probably unrelated and I can be tackle seperately.

![frame-view](https://github.com/gui-cs/Terminal.Gui/assets/31306100/45b9a1c0-5c3d-45eb-aee4-2ace2688557a)

**Comment by tig** [comment]
This has nothing to do, specifically with FrameView. All FrameView is now is a View with BorderStyle set. 

See Arrangement (which is only partially implemented). 

I'm not sure how TGD does size/drag. 

But we want size/drag built into TG (see v2 Issue on it). What we shouldn't have is TG and TGD having two mechanisms. TG should be able to provide TGD with something that works.

**Comment by tig** [comment]
See https://github.com/gui-cs/Terminal.Gui/issues/2537
